src/utrition/src/components/imageupload/imageupload.js

```diff
@@ -14,11 +17,81 @@ const ImageUpload = () => {
     upload_type: "image",
   };
 
+  // New header
+  const headers_confirm = {
+    upload_type: "confirm",
+  };
+
+  
   const handleClick = (event) => {
     fileInput.current.click();
   };
 
+  // Doesn't send any response, set food to 'correct', close confirmation
```

Comment:
No responses sent if food is correct, just update the corresponding state variables, and close menus